compiler/rustc_trait_selection/src/opaque_types.rs

```diff
@@ -46,6 +46,7 @@ pub struct OpaqueTypeDecl<'tcx> {
     /// type Foo = impl Baz;
     /// fn bar() -> Foo {
     /// //          ^^^ This is the span we are looking for!
+    /// # }
```

Comment:
Why this change btw?

Reply:
Without it the build fails because of -Dwarnings, see https://github.com/rust-lang/rust/pull/79816#issuecomment-748532010

Reply:
Then use `}` instead of `# }`, there is not much point into hiding this paren.

Reply:
Done :)